_docs/README.md

```diff
@@ -36,6 +36,8 @@ In order to develop for Grüvee you are going to need to have a few things insta
 
 Now we are at the GOOD stuff. Time to get Grüvee running locally. Make sure you have the [prerequisites](#prerequisites) installed. Open up **two instances** of your favorite terminal/command prompt and navigate to the root directory of the Grüvee repository for both of them. One is going to run your React Native bundle server and the other will start the Grüvee iOS or Android app.
 
+At the time of writing, if you run into any issues try switching to `release-1.0` branch
+
```

Comment:
I think we should have it as a prereq to switch to `release-1.0` off the bat as thats our most up to date branch on! Thoughts?

Reply:
Sure, sounds good!